### PR TITLE
Remove entry comments and reviews

### PR DESCRIPTION
**Pull request by tnagorra**
Depends on https://github.com/idmc-labs/helix-server/pull/347

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

### CODE REVIEW

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/views/Report/ReportCrisisTable/index.tsx [hunk 1]
```diff
@@ -211,11 +205,7 @@ function ReportCrisisTable(props: ReportCrisisProps) {
                     sortable: false,
                 },
                 cellRenderer: StackedProgressCell,
-                cellRendererParams: (_, datum) => ({
-                    signedOff: datum.reviewCount?.signedOffCount,
-                    reviewCompleted: datum.reviewCount?.reviewCompleteCount,
-                    underReview: datum.reviewCount?.underReviewCount,
-                    toBeReviewed: datum.reviewCount?.toBeReviewedCount,
+                cellRendererParams: () => ({
                 }),
```
[the remark this answers is not in the document]
Comment (tnagorra) [reply src/views/Report/ReportCrisisTable/index.tsx:209]: Yes sir!

File: src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/EventsAction/index.tsx [hunk 2]
```diff
@@ -64,26 +76,108 @@ function EventActionCell(props: ActionProps) {
...
+        [onAssignYourself, id],
+    );
+    const handleClearAssignmentYourselfButtonClick = useCallback(
+        () => {
+            if (onClearAssignmentYourself) {
+                onClearAssignmentYourself(id);
+            }
+        },
+        [onClearAssignmentYourself, id],
+    );
```
[the remark this answers is not in the document]
Comment (tnagorra) [reply src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/EventsAction/index.tsx:110]: Pika Pika

[the remark this answers is not in the document]
Comment (tnagorra) [reply src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/EventsAction/index.tsx:110]: Can't use name={id} syntax for confirmation buttons

File: src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/index.tsx [hunk 3]
```diff
@@ -192,6 +257,13 @@ function NudeEventTable(props: EventsProps) {
         [showAddEventModal],
     );
 
+    const handleEventAssigneeChange = useCallback(
+        (id: string) => {
+            showEventAssigneeChangeModal(id);
+        },
+        [showEventAssigneeChangeModal],
+    );
+
```
[the remark this answers is not in the document]
Comment (tnagorra) [reply src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/index.tsx:266]: pi pi

**Pull request merged**